rampwf/score_types/detection/precision_recall.py

```diff
@@ -189,7 +195,12 @@ def mad_center(y_true, y_pred, matches=None, iou_threshold=0.5,
     loc_true, loc_pred = _locate_matches(
         y_true, y_pred, matches, iou_threshold=iou_threshold)
 
-    d = np.sqrt((loc_pred[:, 0] - loc_true[:, 0]) ** 2 + (
-        loc_pred[:, 1] - loc_true[:, 1]) ** 2)
+    x_true, y_true, rad_true = loc_true.T
+    x_pred, y_pred, _ = loc_pred.T
+
+    if not x_true:
+        return np.inf
```

Comment:
I am wondering a bit, is this better than NaN ?

Reply:
I just wanted to conform to the definition of the score and its 'maximum value'.

Reply:
OK, but starting from that definition: [0 - np.inf] is in principle the range of *valid* scores (the lower the bigger the difference in radius, you go towards infinity), while here there is no actual value to calculate. It is just that this score is not defined when there are no matches. And not defined is not necessarily the same as np.inf

But anyhow, doesn't really matter, as both mean there is something really wrong with the model :-)